flump/__init__.py

```diff
@@ -81,35 +81,21 @@ def register_flump_view(self, view_class, url):
         # on both anyway.
         url = url.rstrip('/')
 
-        get_many_func = (
-            view_func if HttpMethods.GET_MANY < methods else _meth_not_allowed
```

Comment:
I'm not sure what `<` mean. Is it short for `s.issubset(t)` and equivalent of ` s <= t`. Is it pyton3 syntax?